doc/api/string_decoder.md

```diff
@@ -63,7 +63,7 @@ Creates a new `StringDecoder` instance.
 added: v0.9.3
 -->
 
-* `buffer` {Buffer|TypedArray|DataView} A `Buffer`, or `TypedArray`, or
+* `buffer` {string|Buffer|TypedArray|DataView} A `Buffer`, or `TypedArray`, or
```

Comment:
The parameter description that follows also needs updating.

Reply:
@mscdex Thank you for your comment. I updated it.